server/mediaserver_test.go

```diff
@@ -1226,6 +1223,10 @@ func TestRegisterConnection(t *testing.T) {
 }
 
 func TestBroadcastSessionManagerWithStreamStartStop(t *testing.T) {
+	goleakOptions := common.IgnoreRoutines()
+	// allow enough time for the transcode end goroutines to finish
+	goleakOptions = append(goleakOptions, goleak.MaxSleepInterval(5*time.Second), goleak.MaxRetryAttempts(1000))
+	defer goleak.VerifyNone(t, goleakOptions...)
```

Comment:
Why 1000 retry attempts and 5 seconds delay (I believe it's a delay between attempts, not overall)? I think, the `EndTranscodingSession` goroutine will finish instantly here. Also maybe pass options to `VerifyNone` call directly?

Reply:
Thanks @cyberj0g. Unfortunately for the tests the communication for the `EndTranscodingSession` call isn't set up so it simply times out after the 3 second [timeout](https://github.com/livepeer/go-livepeer/blob/d8ad5c7/server/rpc.go#L37), so that's the reason for waiting. And the other annoying thing is the goleak library starts off with very small microsecond pauses between retries so that's the reason for the high number. 
I've looked into a lot of different ways of abstracting this away for the tests without success, I think it'll have to wait for a later PR.